internals/cli/cli.go

```diff
@@ -80,9 +80,9 @@ var commands []*cmdInfo
 // debugCommands holds information about all debug commands.
 var debugCommands []*cmdInfo
 
-// addCommand replaces parser.addCommand() in a way that is compatible with
+// AddCommand replaces parser.AddCommand() in a way that is compatible with
```

Comment:
Extremely minor point but parser.addCommand vs parser.AddCommand for the original one?